### ISSUE TITLE
Error during start: Can't open section 'languages'

### ISSUE THREAD

**Issue by aregic** [OpenXRay/xray-16#286]
I'm trying to run the game on Linux Mint over a GOG version of Stalker:COP. I'm having the following error on start:

```
Available render modes[2]:
renderer_gl
Executing config-script "user.ltx"...
! Cannot open script file [user.ltx]
Loading DLL: xrGame
 
FATAL ERROR
 
[error] Expression    : <no expression>
[error] Function      : r_section
[error] File          : /media/regic/2nd_SD/linux/COP/xray-16/src/xrCore/xr_ini.cpp
[error] Line          : 704
[error] Description   : fatal error
[error] Arguments     : Can't open section 'languages' (only 'leader' avail). Please attach [*.ini_log] file to your bug report
```

I don't know where the *.ini_log is located, but I attach the only log I've found: 
[openxray_asdasd.log](https://github.com/OpenXRay/xray-16/files/2612762/openxray_asdasd.log)

**Comment by Xottab-DUTY** [comment]
Did you copy `res/gamedata` to you COP folder?

**Comment by PsiKnapp** [comment]
I was having the languages problem myself. I think that step of copying over the contents of the res folder into your stalker directory should be included in the `build.txt` if it's a necessary step to run properly. This solved the issue for me earlier though.

**Comment by aregic** [comment]
Copying gamedata does solve this problem for me.

**Comment by Yaksinikos** [comment]
Maybe this is helpful;
҂1 Running the game, you have to execute "./bin-linux-dbg/xr_3da.sh -fsltx ../fsgame.ltx" from inside of COP-folder NOT from inside of "bin-linux-dbg"!
Trying to start it using "./xr_3da.sh -fsltx ../fsgame.ltx" from inside of "bin-linux-dbg" leads to "fatal error" -> Can't open section 'languages'!

҂2 you have to copy the folder "gamedata" from the build git-repo to your COP folder!

**Comment by Xottab-DUTY** [comment]
Fixed with b76e6f655da06d82e116253c708f5d36531971f0
